apps/web/tsconfig.server.json

```diff
@@ -7,5 +7,5 @@
     "tsBuildInfoFile": "../../tmp/buildcache/apps/web/server",
     "types": ["node"]
   },
-  "include": ["server.ts"] //"include": ["server.ts", "**/*.js", "**/*.ts", "**/*.tsx", "next-env.d.ts"]
```

Comment:
If we remove change this then the build fails on the jest test. Anyone know why? @kksteini @kksteini